common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala

```diff
@@ -41,13 +51,13 @@ object Mappings {
     nonDynamicObj(
       // TODO: extract these to a configuration setting
       "picdarUrn" -> standardAnalysedString
-    )
+    )()
```

Comment:
Bit :hankey: isn't it? Can `attrs` not be made into a regular trailing parameter with default value because of the varargs `*` first parameter? Is it worth defining another function similar to `nonDynamicObj` that only takes one argument so you don't have to pollute every single use of it with an extra `()` when that extra param is only used once?